solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/LTRRescorer.java

```diff
@@ -118,28 +122,53 @@ public int compare(ScoreDoc a, ScoreDoc b) {
 
     topN = Math.min(topN, firstPassTopDocs.totalHits);
     final ScoreDoc[] reranked = new ScoreDoc[topN];
-    String[] featureNames;
-    float[] featureValues;
-    boolean[] featuresUsed;
-
     final List<LeafReaderContext> leaves = searcher.getIndexReader().leaves();
+    final ModelWeight modelWeight = (ModelWeight) searcher
+        .createNormalizedWeight(reRankModel, true);
+
+    // FIXME: I dislike that we have no gaurentee this is actually a
```

Comment:
Did I write this comment? Heh I feel like I wrote this comment. Can we make a super minor jira ticket with like priority 0 for this?